### ISSUE TITLE
Do we really need services?

### ISSUE THREAD

**Issue by vahidvdn** [amehat/nestjs-cqrs-saga#15]
I guess in CQRS we don't need the services at all, since services usually are used for having database queries and we are delegating this to the command and query executors, having services doesn't make too much sense. Any idea?

**Comment by amehat** [comment]
Absolutely, what corresponds to a service in an MVC model, corresponds to a Command or a Query in a CQRS model. However, nothing prevents having services for responsibilities that can be called in commands or query.
